src/MapsApp.WPF/MapsApp.WPF.csproj

```diff
@@ -210,7 +196,14 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <PackageReference Include="Esri.ArcGISRuntime.WPF">
+      <Version>100.4.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.Windows.Interactivity.WPF">
+      <Version>2.0.20525</Version>
+    </PackageReference>
```

Comment:
Use this official Microsoft package instead:
```xml
    <PackageReference Include="Microsoft.Xaml.Behaviors.Wpf">
      <Version>1.0.1</Version>
    </PackageReference>
```
Note you'll have to change the namespace declarations for `http://schemas.microsoft.com/expression/2010/interactivity` and `http://schemas.microsoft.com/expression/2010/interactions` to a single
`http://schemas.microsoft.com/xaml/behaviors` namespace instead.

Reply:
See https://github.com/Esri/data-collection-dotnet/pull/9

Reply:
...You can also undo this commit now: https://github.com/Esri/maps-app-dotnet/commit/2737ee8be7f4b95679c8369f2c0923f36bb5bf97